### PR TITLE
Added required notebook

### PR DESCRIPTION
**Pull request by souravg-db**

### CODE REVIEW

File: examples/update_owner_of_data_objects.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  except Exception as exception: 
+    print(f"  Exception occurred while updating owner: {exception}")
+
+# COMMAND ----------
+
+dx.from_tables(from_table_statement).map(update_owner)
+
+# COMMAND ----------
+
+(dx.from_tables(from_table_statement)
```
Comment (edurdevic) [inline examples/update_owner_of_data_objects.py:81]: I think we don't need this one.

**Review by edurdevic — CHANGES_REQUESTED** [review]
We are updating the owner twice now.
Please remove one of the two triggers

File: examples/update_owner_of_data_objects.py (hunk 1 above)
Comment (souravg-db) [reply examples/update_owner_of_data_objects.py]: Removed

**Review by edurdevic — APPROVED** [review]
LGTM

**Pull request merged**